src/huntsman/pocs/utils/flats.py

```diff
@@ -10,6 +10,28 @@
 from huntsman.pocs.utils.logger import logger as LOGGER
 
 
+from astropy.coordinates import get_sun
+from astropy.coordinates import AltAz
+
+
+def get_flat_field_altaz(location):
+    """ Return the alt/az opposite the Sun now at a given location.
+    Args:
+        location (astropy.coordinates.EarthLocation): The location.
+    Returns:
+        astropy.coordinates.AltAz: The alt/az opposite the Sun.
+    """
+    time_now = current_time()
+    frame = AltAz(obstime=time_now, location=location)
+    altaz_sun = get_sun(time_now).transform_to(frame)
+
+    # We want the anti-solar direction
+    alt = -1 * altaz_sun.alt  # Alt between -90 and 90 deg
```

Comment:
I don't think this is what we want for the alt.  "The null point for twilight flats is on the solar circle near the zenith, offset toward the antisolar horizon by 0 to 40 deg."

Until we have the time to do our investigation of this I think we should just stick with an alt of 70 degrees as before

Reply:
Makes sense!